readme.md

```diff
@@ -139,6 +139,13 @@ Properties from `options` will override properties in the parsed `url`.
 
 If no protocol is specified, it will throw a `TypeError`.
 
+**Note:** The query string is **not** parsed as search params. Example:
+
+```
+got('https://example.com/?query=a b'); //=> https://example.com/?query=a%20b
+got('https://example.com/', {searchParams: {query: 'a b'}}); //=> https://example.com/?query=a+b
```

Comment:
This is not clear enough. It should also show an example with both `?query` and `{searchParams..`

Reply:
Fixed